src/main/resources/EventDatabase.json

```diff
@@ -22,6 +22,13 @@
       "item": "Potion",
       "text": "You discovered a potion on the ground!"
     },
+    {
+      "name": "test 4",
+      "type": "Encounter",
+      "encounter-type": "Quest",
```

Comment:
here should just be type: "Quest" instead of repurposing Encounter to also work with Quests